proto/sentry_protos/snuba/v1alpha/endpoint_tags_list.proto

```diff
@@ -3,24 +3,31 @@ syntax = "proto3";
 package sentry_protos.snuba.v1alpha;
 
 import "sentry_protos/snuba/v1alpha/request_common.proto";
+import "sentry_protos/snuba/v1alpha/trace_item_attribute.proto";
 
 //A request for "which tags are available for these projects between these dates" - used for things like autocompletion
-message TagsListRequest {
+message TraceItemAttributesRequest {
   RequestMeta meta = 1;
   uint32 limit = 2;
   uint32 offset = 3;
+  AttributeKey.Type type = 5;
 }
 
-message TagsListResponse {
-  enum Type {
-    TYPE_UNSPECIFIED = 0;
-    TYPE_STRING = 1;
-    TYPE_NUMBER = 2;
-  }
-
+message TraceItemAttributesResponse {
   message Tag {
     string name = 1;
-    Type type = 2;
+    AttributeKey.Type type = 2;
   }
   repeated Tag tags = 1;
 }
+
+message AttributeValuesRequest {
+  RequestMeta meta = 1;
+  string name = 3;
```

Comment:
Ahh actually this is missing an argument for a substring match. i.e. if an user starts typing a value, we should pass it through and only return values that contain that substring.

Reply:
Done